bayesian_mapper:
  ros__parameters:
    width: 50.0
    height: 50.0
    resoulution: 0.05
    laser_scan_topic: "/scan"
    use_sim_time: False

vfh_follower:
  ros__parameters:
    linear_speed: 0.2
    angular_speed_max: 1.0
    robot_radius: 0.25
    safety_radius: 0.25
    goal_tolerance: 0.2
    w_goal: 1.0
    w_smooth: 0.3
    w_clear: 0.5
    ground_truth_topic: "/odom"
    laser_scan_topic: "/scan"
    use_sim_time: False
  
map_loader:
  ros__parameters:
    yaml_path: "/home/magraz/rob599_ws/src/hw3/maps/office.yaml"